geoportal/c2cgeoportal_geoportal/scaffolds/create/+dot+github/workflows/main.yaml_tmpl

```diff
@@ -68,16 +68,22 @@ jobs:
 
       - name: Build geoportal
         run: ./build --geoportal
+        if: hashFiles('geoportal/Dockerfile') != ''
```

Comment:
Shouldn't these kind of changes be in a separate PR?

Reply:
Yes I can :-)